.gitlab-ci.yml

```diff
@@ -41,6 +41,31 @@ build:docker:
     - when: never
   script: ["true"]
 
+build:archive:
+  rules:
+    - when: manual
```

Comment:
Why you want this job to be manual? I know it might look like a waste to always run it, but I bet you don't want it to break unnoticed and then not be able to build a package from a tagged commit...

Reply:
I run it manually right now and... [bingo](https://gitlab.com/Northern.tech/northerntechhq/nt-connect/-/jobs/5960370814).